.github/workflows/workflow-linter.yml

```diff
@@ -1,12 +0,0 @@
----
-name: Workflow linter
-
-on:
-  pull_request:
-    paths:
-      - .github/workflows/**
-
-jobs:
-  call-workflow:
-    name: Lint
-    uses: bitwarden/gh-actions/.github/workflows/workflow-linter.yml@main
```

Comment:
Why remove the linting workflow?

Reply:
This is from an old version of the template and shouldn't be in individual repos. It's executed as an organizational workflow now. I remove this every time I see it.